### PR TITLE
Fix examples: add rel attribute and fix typo

### PR DESCRIPTION
**Pull request by ChhikaraBRUH**
- Add rel="stylesheet" attribute
- Fix "unminify" typo to "minify"

### CODE REVIEW

File: resources/examples/additional-parameters.html [hunk 1]
```diff
@@ -1 +1 @@
-<link href="https://cdn.curlwind.com?classes=p-*:sm|md,m-*:hover">
+<link rel="stylesheet" href="https://cdn.curlwind.com?classes=p-*:sm|md,m-*:hover">
```
Comment (stevebauman) [inline resources/examples/additional-parameters.html:1]: I don't want to include this attribute as it obfuscates the examples

They should be as plain as possible. Anyone pasting the link in can just add them afterwards, as it'll work normally with or without `rel="stylesheet"`

**Review by stevebauman — COMMENTED** [review]
Thanks for the PR! Can you remove the `rel` attributes and leave the minify update and I'll merge?

File: resources/examples/additional-parameters.html (hunk 1 above)
Comment (stevebauman) [reply resources/examples/additional-parameters.html]: Nevermind, done! 👍

**Comment by stevebauman** [comment]
Sorry you were right -- we need the `rel=stylesheet` attribute. My mistake! I'll push these changes now 👍

**Comment by ChhikaraBRUH** [comment]
I'm glad :)

**Pull request merged**